pinot-core/src/main/java/org/apache/pinot/core/operator/AcquireReleaseColumnsSegmentOperator.java

```diff
@@ -81,6 +86,12 @@ public void release() {
     _indexSegment.release(_fetchContext);
   }
 
+  @Override
+  public void prepareForExplainPlan(ExplainPlanRows explainPlanRows) {
+    materializeChildOperator();
+    acquire();
```

Comment:
Should acquire happen before materializeChildOperator?

Reply:
My bad. Yes.